### PR TITLE
Do not install ingest-attachment in xpack qa tests

### PR DESCRIPTION
**Pull request by jkakavas**
Don't attempt to install ingest-attachment in smoke-test-plugins
and smoke-test-plugins-ssl.

See 12b24bfa57ae3b282427298cffc735ab9fe5868f, x-pack/qa tests were
missed in that change.

### CODE REVIEW

File: x-pack/qa/smoke-test-plugins-ssl/build.gradle [hunk 1]
```diff
@@ -68,6 +69,10 @@ testClusters.integTest {
   user username: "monitoring_agent", password: "x-pack-test-password", role: "remote_monitoring_agent"
 
   project(':plugins').getChildProjects().each { pluginName, pluginProject ->
+    if (BuildParams.inFipsJvm && pluginName == "ingest-attachment"){
```
Comment (jasontedor) [inline x-pack/qa/smoke-test-plugins-ssl/build.gradle]:
```suggestion
    if (BuildParams.inFipsJvm && pluginName == "ingest-attachment") {
```

File: x-pack/qa/smoke-test-plugins/build.gradle [hunk 2]
```diff
@@ -15,6 +16,10 @@ testClusters.integTest {
   setting 'xpack.license.self_generated.type', 'trial'
   user username: "test_user", password: "x-pack-test-password"
   project(':plugins').getChildProjects().each { pluginName, pluginProject ->
+    if (BuildParams.inFipsJvm && pluginName == "ingest-attachment"){
```
Comment (jasontedor) [inline x-pack/qa/smoke-test-plugins/build.gradle]:
```suggestion
    if (BuildParams.inFipsJvm && pluginName == "ingest-attachment") {
```

File: x-pack/qa/smoke-test-plugins-ssl/build.gradle [hunk 3]
```diff
@@ -68,6 +69,10 @@ testClusters.integTest {
...
+      //Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
```
Comment (jasontedor) [inline x-pack/qa/smoke-test-plugins-ssl/build.gradle]:
```suggestion
      // Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
```

File: x-pack/qa/smoke-test-plugins/build.gradle [hunk 4]
```diff
@@ -15,6 +16,10 @@ testClusters.integTest {
...
+      //Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
```
Comment (jasontedor) [inline x-pack/qa/smoke-test-plugins/build.gradle]:
```suggestion
      // Do not attempt to install ingest-attachment in FIPS 140 as it is not supported (it depends on non-FIPS BouncyCastle)
```

**Review by jasontedor — APPROVED** [review]
I left some nits, otherwise LGTM.

**Comment by jkakavas** [comment]
@elasticmachine run elasticsearch-ci/default-distro

**Comment by jkakavas** [comment]
@elasticmachine run elasticsearch-ci/default-distro

**Comment by jkakavas** [comment]
@elasticmachine update branch

**Pull request merged**